survey-runner.tf

```diff
@@ -58,7 +59,7 @@ module "eq-ecs" {
 }
 
 module "survey-runner-on-ecs" {
-  source                  = "github.com/ONSdigital/eq-survey-runner-deploy"
+  source                  = "github.com/ONSdigital/eq-survey-runner-deploy?ref=eq-1016-add-preprod-variable-to-deploy"
```

Comment:
You don't need this branch ref now its merged.

Reply:
@MIslam92 if you remove this i can tick it.

Reply:
?? I've removed this ??